README.md

```diff
@@ -10,7 +10,46 @@ First, you'll need to configure a database. Once this project is deployed, we'll
 
 ### Windows setup
 
-Coming Soon!
+## Chocolatey and Postgres: (Needs Testing)
```

Comment:
Can you please make these headings four `#`? They should be smaller than `Windows setup`

Reply:
fixed